### DISCUSSION TITLE
Feature Request: notion editor for vue

### DISCUSSION THREAD

**Discussion by nero23333** [ueberdosis/tiptap#5125] [Feature Requests]
### Description

Please, I realy need a vue notion editor template ,I think there are many people wants it too.

### Use Case

Build a note editor for my website

### Type

New extension

**Comment by johnpuddephatt** [comment]
I've built something similar in Vue: https://github.com/johnpuddephatt/gutentap/

**Reply by nperez0111** [reply]
A very nice editor indeed!

**Comment by nperez0111** [comment]
Appreciate the enthusiasm, but I don't believe that we have the bandwidth for something like this right now
